services/notification/template.yaml

```diff
@@ -30,7 +30,7 @@ Resources:
     Properties:
       CodeUri:
         Bucket: !Sub 'cz-provision-account-${AWS::Region}'
-        Key: latest/services/notification.zip
+        Key: v1.0.dev/services/notification.zip
```

Comment:
latest or dev?

Reply:
needs to be `latest`; i was just doing some testing ... it's back now